.github/workflows/build-and-test.yml

```diff
@@ -1,202 +1,192 @@
 name: Build
 
 on:
-   push:
-       branches:
-           - master
-       tags:
-           - "v*"
-   pull_request:
-       branches:
-          - master
+  push:
+    branches:
+      - master
+    tags:
+      - "v*"
+  pull_request:
+    branches:
+      - master
 jobs:
-    Build:
-        strategy:
-            matrix:
-                options:
-                    - os: ubuntu-latest
-                      framework: net6.0
-                      sdk: 6.0.x
-                      sdk-preview: true
-                      runtime: -x64
-                      codecov: false
-                    - os: macos-latest
-                      framework: net6.0
-                      sdk: 6.0.x
-                      sdk-preview: true
-                      runtime: -x64
-                      codecov: false
-                    - os: windows-latest
-                      framework: net6.0
-                      sdk: 6.0.x
-                      sdk-preview: true
-                      runtime: -x64
-                      codecov: false
-                    - os: ubuntu-latest
-                      framework: net5.0
-                      runtime: -x64
-                      codecov: false
-                    - os: macos-latest
-                      framework: net5.0
-                      runtime: -x64
-                      codecov: false
-                    - os: windows-latest
-                      framework: net5.0
-                      runtime: -x64
-                      codecov: false
-                    - os: ubuntu-latest
-                      framework: netcoreapp3.1
-                      runtime: -x64
-                      codecov: true
```

Comment:
we also loose the netcoreapp3.1 PR tests with this change, is that intentional? 
I would have thought we we just want to set `codecov: false` instead of removing the whole target?

Reply:
Whoops! You’re right; will fix

Reply:
Done